CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
 - Added two new properties to mapping json files. The `colors: [<hsvHueValue1>, <hsvHueValue2>, ...]` property can be used to specify up to 256 custom colors for the first 256 equivalence classes of the mapping. The `hideUnmappedIds: <true|false>` property indicates whether segments that were not mapped should be rendered transparently or not. [#2965](https://github.com/scalableminds/webknossos/pull/2965)
 - Added a button for refreshing the dataset in the backend cache. [#2975](https://github.com/scalableminds/webknossos/pull/2975)
+- Added a popover to the explorative annotations view that displays when hovered over the name of an annotation. [#3035](https://github.com/scalableminds/webknossos/pull/3035)
```

Comment:
I think, this is a bit too detailed. I'd change it to "Added the possibility to see the description of a tracing within the dashboard."